src/modules/map/components/WarningRibbon.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+    <div v-if="hasWarningRibbon" class="corner-ribbon" data-cy="warning-ribbon">TEST</div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+
+export default {
+  computed: {
+      ...mapGetters([
+            'hasWarningRibbon',
+      ]),
+
+    }
+
+}
+</script>
+
+<style lang="scss" scoped>
+@import 'src/scss/media-query.mixin';
+@import 'src/scss/variables-admin';
+@import 'src/scss/variables';
+
+/* Corner ribbons, from http://codepen.io/eode9/pen/twkKm
+* mobile: ribbon top left, desktop: bottom left
+*/
+
+.corner-ribbon {
+  position: absolute;
+  /* top-left */
+  top: 78px;
+  bottom: auto;
+  left: -50px;
+  width: 200px;
+  transform: rotate(-45deg);
+  z-index: $zindex-warning;
+  background: $red;
+  color: white;
+  text-align: center;
+  line-height: 50px;
+  letter-spacing: 1px;
+  font-weight: bold;
+  font-size: 2em;
```

Comment:
I personally prefer without the font-size (maybe matter of taste):

With font-size `2em`:
![image](https://user-images.githubusercontent.com/67745584/208600262-ab8877f5-4521-4a12-bf2e-579a8e27994d.png)

Without font-size:
![image](https://user-images.githubusercontent.com/67745584/208600305-4bc8f5c6-fb12-47a9-b74e-d963a5106b76.png)

Without looks more similar to the current ribbon banner on test.map.geo.admin.ch

Reply:
OK, as you want.